mne/time_frequency/tests/test_spectrum.py

```diff
@@ -206,15 +205,8 @@ def _agg_helper(df, weights, group_cols):
 
 
 @pytest.mark.parametrize("long_format", (False, True))
-@pytest.mark.parametrize(
```

Comment:
Rather than removing tests (which means we could break support within this release cycle) you could instead keep the tests as they are except adding to the two tests that need it:
```
@pytest.mark.filterwarnings("ignore:Complex output support.*:RuntimeWarning")
```

Reply:
done in [fe6ac74](https://github.com/mne-tools/mne-python/pull/11978/commits/fe6ac747ee239eb086b839ce89766f85e3543d81)